docs/topics/feed-exports.rst

```diff
@@ -429,3 +430,41 @@ format in :setting:`FEED_EXPORTERS`. E.g., to disable the built-in CSV exporter
 .. _Amazon S3: https://aws.amazon.com/s3/
 .. _botocore: https://github.com/boto/botocore
 .. _Canned ACL: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl
+
+.. setting:: FEED_STORAGE_BATCH_ITEM_COUNT
+
+FEED_STORAGE_BATCH_ITEM_COUNT
+-----------------------------
+Default: ``None``
```

Comment:
I agree with @ejulio:

```suggestion
Default: ``0``
```

And we could use `BaseSettings.getint` instead of `BaseSettings.get` when querying this setting.